includes/Classifai/Features/ExcerptGeneration.php

```diff
@@ -251,13 +251,6 @@ public function enqueue_admin_assets( string $hook_suffix ) {
 					);
 				}
 			}
-
-			wp_enqueue_style(
-				'classifai-language-processing-style',
-				CLASSIFAI_PLUGIN_URL . 'dist/language-processing.css',
-				[],
-				get_asset_info( 'language-processing', 'version' ),
-			);
```

Comment:
It seems the language processing CSS file contains some CSS of the excerpt generation model. Removing this minor break spacing above the button. 

![image](https://github.com/user-attachments/assets/3cd9d7c2-1b88-4360-82e3-f7c4e3e7b39d)